interval/bounds.go

```diff
@@ -0,0 +1,57 @@
+package interval
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/influxdata/flux/values"
+)
+
+const (
+	MaxTime = math.MaxInt64
+	MinTime = math.MinInt64
+)
+
+type Bounds struct {
+	start values.Time
+	stop  values.Time
+	// As additional windows are added to or subtracted from the initial bounds, index keeps track of how many
```

Comment:
godoc comments should start with the name of the variable first. So something like:

```
// index keeps track of how many windows have been added or subtracted from the initial bounds etc.
```

A common formatting is to have a newline between the godoc comment and the previous line. Can you also document the other two fields even if they are kind of obvious? Even if it's obvious, it's useful because then there's no ambiguity for things like "is this time boundary inclusive or exclusive?"